Dockerfile

```diff
@@ -2,82 +2,63 @@
 # Zulip development environment image and use
 # tools/build-release-tarball to generate a production release tarball
 # from the provided Git ref.
-FROM ubuntu:xenial-20171114
-LABEL maintainer="Alexander Trost <galexrt@googlemail.com>"
+FROM ubuntu:18.04 as base
 
-# You can specify these in docker-compose.yml or with
-#   docker build --build-args "ZULIP_GIT_REF=git_branch_name" .
-ARG ZULIP_GIT_URL=https://github.com/zulip/zulip.git
-ARG ZULIP_GIT_REF=2.0.4
-ARG CUSTOM_CA_CERTIFICATES=
+# Set up working locales and upgrade the base image
+ENV LANG="C.UTF-8"
 
-SHELL ["/bin/bash", "-xuo", "pipefail", "-c"]
+ARG UBUNTU_MIRROR
 
-# First, we setup working locales
-RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends && \
-    echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends && \
+RUN { [ ! "$UBUNTU_MIRROR" ] || sed -i "s|http://\(\w*\.\)*archive\.ubuntu\.com/ubuntu/\? |$UBUNTU_MIRROR |" /etc/apt/sources.list; } && \
     apt-get -q update && \
-    apt-get -q install locales && \
-    locale-gen en_US.UTF-8
+    apt-get -q dist-upgrade -y && \
+    DEBIAN_FRONTEND=noninteractive \
+        apt-get -q install --no-install-recommends -y ca-certificates git locales lsb-release python3 sudo tzdata
 
-ENV LANG="en_US.UTF-8" \
-    LANGUAGE="en_US:en" \
-    LC_ALL="en_US.UTF-8"
```

Comment:
Why is the locale code no longer needed?  It certainly was on Xenial; is the change that Bionic has a better locale setup, or something else?

Reply:
I replaced it with `LANG="C.UTF-8"`, which is provided by default and works fine.

Reply:
Ahh, OK, C.UTF-8, not "C".  That makes sense.